website_snippet_carousel_product/static/src/js/s_product_carousel_frontend.js

```diff
@@ -0,0 +1,104 @@
+odoo.define("website_snippet_carousel_product.s_product_carousel", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var sAnimation = require("website.content.snippets.animation");
+
+    var _t = core._t;
+
+    sAnimation.registry.js_product_carousel = sAnimation.Class.extend({
+        selector: ".js_product_carousel",
+
+        _refreshAnimations: function () {
+            this.trigger_up("animation_start_demand", {
+                editableMode: false,
+                $target: this.$target.find(".product_carousel_slider"),
+            });
+        },
+
+        /**
+         * @override
+         */
+        start: function () {
+            var self = this;
+            var limit = Number(this.$target.attr("data-products-limit")) || 12;
+            var domain = this.$target.attr("data-domain") || false;
+            var products_per_slide =
+                Number(this.$target.attr("data-products-per-slide")) || 4;
+            var interval = this.$target.attr("data-interval");
+            if (typeof interval === 'undefined') {
+                interval = 5000;
+            } else {
+                interval = Number(interval);
+            }
+
+            // Compatibility with db that saved content inside by mistake
+            this.$target.empty();
+            // Prevent user edition
+            this.$target.attr("contenteditable", "False");
+
+            var def = this._rpc({
+                route: "/website/render_product_carousel",
+                params: {
+                    limit: limit,
+                    domain: JSON.parse(domain),
+                    products_per_slide: products_per_slide,
+                },
+            })
+                .then(
+                    function (object_html) {
+                        var $object_html = $(object_html);
+                        var count = $object_html
+                            .find("input[name='product_count']")
+                            .val();
+                        if (!count) {
+                            self.$target.append(
+                                $("<div/>", {class: "col-md-6 offset-md-3"}).append(
+                                    $("<div/>", {
+                                        class:
+                                            "alert alert-warning alert-dismissible text-center",
+                                        text: _t(
+                                            "No products was found. Make sure you have products published on the website."
+                                        ),
+                                    })
+                                )
+                            );
+                            return;
+                        }
+
+                        self.$target.html($object_html);
+                        self.$target.find('.carousel').carousel({
+                            interval: interval,
```

Comment:
I see more problems than advantages trying to use the attribute instead of js (which in the end is what bootstrap will do).
But, now the interval options works too similar.